src/lithium/reducer.py

```diff
@@ -323,8 +327,16 @@ def processArgs(self, parser, args):
         if not isPowerOfTwo(self.minimizeMin) or not isPowerOfTwo(self.minimizeMax):
             parser.error("Min/Max must be powers of two.")
 
+    @staticmethod
+    def post_round_cb(testcase):  # pylint: missing-param-doc,missing-type-doc,unused-argument
```

Comment:
Ditto about the docstring format here should be [standardised to this style in Lithium](https://github.com/MozillaSecurity/lithium/blob/master/src/lithium/interestingness/crashes.py#L22).

The `missing-param-doc` and `missing-type-doc` pylint error ignores can then be removed.